### ISSUE TITLE
Python packages need to be installed

### ISSUE THREAD

**Issue by Avilir** [bengland2/ocp4_upi_baremetal#8]
Need to update the doc that the packages : asyncio, aiohttp
are need to be installed on the host that using to run the playbooks, since those packages are not part of the python installation.

**Comment by bengland2** [comment]
Why are these packages needed @Avilir ?   For which playbook?   Which RPM or git repo depends on them?    I'm running my deployment without them.   So apparently these packages are not needed to deploy OCP4, which is what this playbook is mostly about.   Just want to understand what the dependency is.  Let's update the procedure to automatically install these, not hard to do.   I want to keep the documented procedure simple.   I found a **python3-aiohttp** package, is this what you meant?

**Comment by Avilir** [comment]
@bengland2 , i had to install the python3-aiohttp since the ~/badfish/badfish.py command failed and asked for the modules.

**Comment by bengland2** [comment]
[Done here](https://github.com/bengland2/ocp4_upi_baremetal/commit/01bcac4976d07fc14432b65e284a7767b3133713) , @Avilir

**Comment by Avilir** [comment]
Grate.

**Comment by ArangoGutierrez** [comment]
should it be a `pip3 install badfish/requirements.txt` step ?

**Comment by bengland2** [comment]
@ArangoGutierrez you're right, [as documented here](https://github.com/redhat-performance/badfish#badfish-standalone-script).  I think you meant 
```
pip install --requirement=requirements.txt
```
I didn't see that there were more dependencies than just aiohttp,  So next chance I get I'll add this command to the part of the playbook that installs badfish and see if that works, I want to do what badfish documentation says to do.
